brahma/data/payload/source/main.c

```diff
@@ -30,7 +30,7 @@ void main()
 	*((unsigned int*)0x080FFFC4) = 0x20046500;
 	*((unsigned int*)0x080FFFD8) = 0;
 
-	unsigned int* buf = 0x20400000;
+	unsigned int* buf = (void*)0x20400000;
```

Comment:
well if change all those `unsigned int` to `u32` and `unsigned char` to `u8`, this would be `(u8*)`. It is only a value pointer cast so it is safe. (* ```void*``` is special..*)